src/pages/patientView/PatientViewPage.tsx

```diff
@@ -78,6 +78,17 @@ import { MakeMobxView } from '../../shared/components/MobxView';
 import ResourceTab from '../../shared/components/resources/ResourceTab';
 import TimelineWrapper from './timeline2/TimelineWrapper';
 import ClinicalEventsTables from './timeline2/ClinicalEventsTables';
+import { OncoKB } from 'react-mutation-mapper';
+import {
+    getSampleNumberClinicalDataValue,
+    OtherBiomarkersQueryType,
+} from 'shared/lib/StoreUtils';
+import { CLINICAL_ATTRIBUTE_ID_ENUM } from 'shared/constants';
+import {
+    MSI_H,
+    OTHER_BIOMARKER_HUGO_SYMBOL,
+    TMB_H,
+} from 'react-mutation-mapper/src/component/oncokb/constants';
```

Comment:
Direct import from package `src` directory is likely to cause a lot of issues. Let's export these constants and do this instead: 
```suggestion
} from 'react-mutation-mapper';
```
We can also consider moving these constants into `cbioportal-utils` if you think it is a better package for them.

Reply:
Not yet, let's keep it under RMM for now.